controller/src/main/java/io/pravega/controller/store/stream/PersistentStreamBase.java

```diff
@@ -1202,33 +1221,22 @@ private TxnStatus handleDataNotFoundException(Throwable ex) {
             }
         });
     }
-
-    @Override
-    public CompletableFuture<Map<UUID, ActiveTxnRecord>> getActiveTxns() {
-        return getCurrentTxns()
-                .thenApply(x -> x.entrySet()
-                                 .stream()
-                                 .collect(toMap(k -> UUID.fromString(k.getKey()),
-                                         v -> ActiveTxnRecord.fromBytes(v.getValue().getData()))));
-    }
-
+    
```

Comment:
Actually, I though on using this as a part of the information printed to describe a Stream from the CLI...

Reply:
The method is not removed. Its implementation is pushed down to the derived classes. 

This method was calling getCurrentTxns and converting String to UUID. 
getCurrentTxns used to be an abstract method in PersistentStreamBase and had store specific implementation.
I removed getCurrentTxns and made the implementation of getActiveTxns in ZkStream and InMemoryStream which now directly deals with UUID rather than returning String which again gets converted to UUID in this class.
Done the same for all txn related apis where we dont do one more scan of the collection to convert string to UUID